tests/fetch_logs.py

```diff
@@ -0,0 +1,38 @@
+import subprocess, shutil, time, os
+
+def run_bash(command):
+    process = subprocess.Popen(command.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+
+
+# if length is negative, returns all lines
+def file_tail(filename, length):
+    with open(filename, "r") as file_to_read:
+        lines = file_to_read.readlines()
+        if length < len(lines) and length >= 0:
+            return "".join(lines[len(lines)-length:])
```

Comment:
Does this print new lines or does it squish everything onto the same line?

Reply:
This prints new lines.